actionpack/test/controller/url_for_integration_test.rb

```diff
@@ -170,6 +170,7 @@ def setup
       ["/posts?foo%5B%5D=bar&foo%5B%5D=baz", [{ controller: "posts", foo: ["bar", "baz"] }]],
       ["/posts?page=2",  [{ controller: "posts", page: 2 }]],
       ["/posts?q%5Bfoo%5D%5Ba%5D=b", [{ controller: "posts", q: { foo: { a: "b" } } }]],
+      ["/posts?_route=1", [{ controller: "posts", _route: 1 }]],
```

Comment:
This make sure we can still pass `_route` is still a valid query parameters. 
This prevents the prioritization of the current route, but it falls back to the old behavior.